authority/src/lib.rs

```diff
@@ -267,7 +267,7 @@ pub mod module {
 		#[pallet::weight(T::WeightInfo::fast_track_scheduled_dispatch())]
 		pub fn fast_track_scheduled_dispatch(
 			origin: OriginFor<T>,
-			initial_origin: T::PalletsOrigin,
+			initial_origin: Box<T::PalletsOrigin>,
```

Comment:
So that decreases the call size from 608 bytes to 24 bytes. Not entirely sure why but numbers don't lie